hashes/adler32.py

```diff
@@ -15,15 +15,15 @@ def adler32(plain_text: str) -> str:
     Itterates and evaluates new value for each character
 
     >>> adler32('Algorithms')
-    363791387
+    '363791387'
 
     >>> adler32('go adler em all')
-    708642122
+    '708642122'
     """
     MOD_ADLER = 65521
     a = 1
     b = 0
     for plain_chr in plain_text:
         a = (a + ord(plain_chr)) % MOD_ADLER
         b = (b + a) % MOD_ADLER
-    return (b << 16) | a
+    return str((b << 16) | a)
```

Comment:
Given that the example in https://en.wikipedia.org/wiki/Adler-32 returns an int, I would prefer that we do so as well.